conference.js

```diff
@@ -1307,7 +1311,7 @@ export default {
                     : 'colibri';
         }
 
-        const nick = APP.settings.getDisplayName();
+        const nick = getSettings(APP.store).displayName;
```

Comment:
this should be retrieved from the local user and not settings, since it could have been overwritten by JWT, right?

Reply:
Yeah, ditto, I think this should be the content of a follow up PR where we clean this up. If you see the original code, that didn't use jwt either, so I didn't want to change it here